uktrade/data-hub-frontend
# Feature/cls2 229 activity show/hide external activity

## Description of change
On the Company > Activity page an `Activity type` filter has been added that allows for filtering on Internal or External activity. Aventri activity should be included with the Internal activity.

## Test instructions

Visit a company's activity page, e.g. [Alphabet](http://localhost:3000/companies/9aacdbb0-32b3-4dc5-a961-24e6e6c57a46/activity) or [One List Corp](http://localhost:3000/companies/375094ac-f79a-43e5-9c88-059a7caa17f0/activity). You should be able to filter the companies activities by type.

## Screenshots

### Before
![image](https://github.com/uktrade/data-hub-frontend/assets/699259/7df6dfc7-c584-4ab5-afa5-e8492f606814)

### After
![image](https://github.com/uktrade/data-hub-frontend/assets/699259/8500bdb8-e53c-4c2f-a2bd-4933300d69dc)

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
